Tests/LinkCommandsTest/README.md

```diff
@@ -0,0 +1,8 @@
+```
+python3 main.py --i="path-to-project"
```

Comment:
Could you please move it out to this repository https://github.com/Software-Analysis-Team/link-commands